runtime/doc/eval.txt

```diff
@@ -6856,6 +6856,21 @@ sinh({expr})						*sinh()*
 			:echo sinh(-0.9)
 <			-1.026517
 
+sockopen({mode}, {address}, {opts})			{Nvim} *sockopen()*
+		Connect a socket to an address. If {mode} is "pipe" then
+		{address} should be the path of a named pipe. If {mode} is
+		"tcp" then {address} should be of the form "host:port" where
+		the host should be an ip adderess or host name, and port the
+		port number. Currently only rpc sockets are supported, so {opts}
+		must be passed with "rpc" set to `v:true`.
```

Comment:
Historically Vim represented booleans simply by using numbers, but more recently `v:true`/`v:false` were added to make it explicit.  It's probably better to reword this like

> Currently only rpc sockets are supported so {opts} must be passed with "rpc" set to |TRUE|.

Unless the code is explicitly checking for the `v:true`/`v:false` special values.